src/materialized/tests/pgwire.rs

```diff
@@ -427,7 +427,7 @@ fn test_record_types() -> Result<(), Box<dyn Error>> {
 fn pg_test_inner(dir: PathBuf) -> Result<(), Box<dyn Error>> {
     // We want a new server per file, so we can't use pgtest::walk.
     datadriven::walk(dir.to_str().unwrap(), |tf| {
-        let server = util::start_server(util::Config::default()).unwrap();
+        let server = util::start_server(util::Config::default().experimental_mode()).unwrap();
```

Comment:
I've had to do something akin to this when e.g. determining the output types of functions; seems reasonable for tests to run in experimental mode all the time and unclear what the downside would be.